app/Http/Controllers/Select/PortGroupController.php

```diff
@@ -38,6 +38,17 @@ protected function baseQuery($request)
         return PortGroup::hasAccess($request->user())->select(['id', 'name']);
     }
 
+    protected function formatResponse($paginator)
+    {
+        // inject 'no default Port group' as choice
+        $general = new PortGroup;
```

Comment:
One problem left, this will show up on every page.  In fact I think mine have the same problem :/
should be able to check the page number from the paginator.

Reply:
is it solved? :-)